tests/components/zeversolar/test_init.py

```diff
@@ -27,34 +26,21 @@ async def test_async_setup_entry_fails(
         pytest.raises(ConfigEntryNotReady),
     ):
         await init.async_setup_entry(hass, config_entry)
-
+        assert config_entry.state is ConfigEntryState.SETUP_RETRY
 
     with (
         patch("homeassistant.components.zeversolar.PLATFORMS", []),
         patch("zeversolar.ZeverSolarClient.get_data", return_value=zeversolar_data),
     ):
-        config_entry.add_to_hass(hass)
-
         result = await hass.config_entries.async_setup(config_entry.entry_id)
         await hass.async_block_till_done()
 
         assert result is True
         assert config_entry.state is ConfigEntryState.LOADED
```

Comment:
There is no result on `async_schedule_reload`, so just check the state (and un-indent)
```suggestion
    assert config_entry.state is ConfigEntryState.LOADED
```